Sources/OktaLogger/Analytics/AppCenterAnalytics.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  AppCenterAnalytics.swift
```

Comment:
Please update header

Reply:
Thanks for catching, we need to change this in the project level file that will automatically create header with copyright